internal/resolvers/twitch/check.go

```diff
@@ -0,0 +1,35 @@
+package twitch
+
+import (
+	"net/url"
+	"regexp"
+
+	"github.com/Chatterino/api/pkg/utils"
+)
+
+var (
+	clipSlugRegex = regexp.MustCompile(`^\/(\w{2,25}\/clip\/)?([a-zA-Z0-9]+(?:-[-\w]{16})?)$`)
```

Comment:
Do we have documentation to confirm we can be this strict with the "extra part" behind the clip slug? I'm referring mostly to the fact that we only consider it valid if it's 16 characters

Reply:
We don't. This regex is just based on many examples and tests that we've done with Supinic and Leppunen when they were updating their smart things to new format of clips.

Reply:
I understand that in the future Twitch might change their clip slug format once again - possibly to make the "extra part" longer than 16 characters, but I think we're good for now, as this new format allows for loads of new clips to be generated, so the probability of this occuring is very low.

Reply:
Would we hit any harmful false positives if we just let it be anything about X amount of characters? i.e. 8 characters

Reply:
I guess nothing bad would happen. If anything, we'd be just less strict about what we pass to load function, but realistically speaking that's no problem at all.
I'm guessing you're willing to change this `{16}` to `+`. If so feel free to make that change and merge the PR.

Reply:
Fair enough - I guess it's more of a "do effort now" vs "do effort then" question, but it's not even sure us changing it to a `+` to solve it so I'll approve it as is